backend/routes/matchRoute.js

```diff
@@ -16,4 +19,26 @@ router.get('/student/:studentId/placements', async (req, res, next) => {
     res.json(placements);
 });
 
+router.delete('/match/:studentId/:placementId', async (req, res, next) => {
+    if( !isNaN(req.params.studentId) && !isNaN(req.params.placementId)){
+        let match = await matchService.deleteMatch(req.params.studentId, req.params.placementId)
+        .catch(error => {
+            res.status(error.code).send(error.message);
+        });
+        res.json(match);
+    } else {
+        res.status(ERR_BAD_REQUEST).send('Your request did not provided valid values for ids. Please try again.');
+    }
+    
+});
+
+router.get('/placement/:placementId/students', async (req, res, next) => {
+    const students = await matchService.getMatchesByPlacementId(req.params.placementId)
```

Comment:
if we put these errors on the delete call " if( !isNaN(req.params.studentId) && !isNaN(req.params.placementId)){ ", we can also put them on the get call, the situation is similar because both calls make use of an id

Reply:
ok